GameOfLife/GameOfLife/Services/FieldOperations.cs

```diff
@@ -0,0 +1,221 @@
+﻿using GameOfLife.Interfaces;
+using GameOfLife.Models;
+using static GameOfLife.StringConstantsModel;
+
+namespace GameOfLife
+{
+    public class FieldOperations : IFieldOperations
+    {
+        private int _coordinateX;
+        private int _coordinateY;
+        private bool _stop = false;
+        private IRender _render;
+        private ILibrary _library;
+        private IInputController _inputProcessor;
+        public int CoordinateX
+        {
+            get => _coordinateX;
+            set => _coordinateX = value;
+        }
+        public int CoordinateY
+        {
+            get => _coordinateY;
+            set => _coordinateY = value;
+        }
+        public bool Stop
```

Comment:
You can have implicit properties:
public bool Stop { get; set; }

Stop is too generic name. Make it more specific.

Reply:
Changed properties to implicit properties. Made 'Stop' more specific.